Meta/Lagom/Tools/CodeGenerators/LibWeb/CMakeLists.txt

```diff
@@ -1,11 +1,6 @@
-add_executable(Generate_CSS_PropertyID_h Generate_CSS_PropertyID_h.cpp)
-add_executable(Generate_CSS_PropertyID_cpp Generate_CSS_PropertyID_cpp.cpp)
-add_executable(Generate_CSS_ValueID_h Generate_CSS_ValueID_h.cpp)
-add_executable(Generate_CSS_ValueID_cpp Generate_CSS_ValueID_cpp.cpp)
-add_executable(WrapperGenerator WrapperGenerator.cpp)
+lagom_tool(Generate_CSS_PropertyID_h   SOURCES Generate_CSS_PropertyID_h.cpp)
```

Comment:
Since this function is taking the `SOURCES` variable from the enclosing scopes, I would recommend putting a `set(SOURCES "")` at the beginning on this lists file, to avoid any chance of accidentally including unrelated sources.

Reply:
Good call, fixed in d6c4ea86018ee722aae5b0b4d34e81e28313b4c6